Makefile

```diff
@@ -8,14 +8,17 @@ install: ## Install development dependencies
 	poetry install
 
 fmt: install ## (Format) - runs black and isort against the codebase (auto triggered on pre-commit)
-	poetry run black ./dpypelines/*
-	poetry run isort ./dpypelines/*
+	poetry run black ./dpytools/*
```

Comment:
This should be `poetry run black ./dpypelines/*`. You can also incorporate the tests folder by doing `poetry run black ./dpypelines/* ./tests/*`.

Reply:
Noted